subt/ros/robot/launch/freyja.launch

```diff
@@ -2,97 +2,61 @@
 <launch>
   <node name="$(arg robot_name)_proxy" pkg="proxy" type="ros_proxy" args="$(arg robot_name)" output="screen">
     <remap from="$(arg robot_name)/front_scan" to="$(arg robot_name)/scan_front"/>
-    <remap from="$(arg robot_name)/image_raw/compressed" to="$(arg robot_name)/camera_front/image_raw/compressed"/>
+    <remap from="$(arg robot_name)/image_raw/compressed" to="$(arg robot_name)/rgbd_front/image_raw/compressed"/>
```

Comment:
When I tried to do the same change in #719, it did not work - I got some error thrown from `rosmsg`. I wonder why it is now working...